packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/Table/index.js

```diff
@@ -83,7 +85,19 @@ const Table = ({
               <Td>
                 {token.lastUsedAt && (
                   <Typography textColor="neutral800">
-                    <RelativeTime timestamp={new Date(token.lastUsedAt)} />
+                    <RelativeTime
+                      timestamp={new Date(token.lastUsedAt)}
```

Comment:
I wonder if converting the date inside the `RelativeTime` component is cleaner, checking if it's a string or if it's already a date. In the end, the component is responsible for the logic of transforming the date into a readable format, so why not encapsulate all the logic inside it?

Reply:
imho this should have been either `children` or `value` and i would always pass a Date because it's inline with what we do with DateTimePicker and DatePicker 👍🏼

Reply:
Nice! I didn't know that 😅 Thanks!